packages/frontend-api/src/Model/Error/InvalidArgumentUserError.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Error;
+
+use GraphQL\Error\UserError;
+
+class InvalidArgumentUserError extends UserError implements UserErrorWithCodeInterface
+{
+    protected const CODE = 'invalid-argument';
```

Comment:
Most of codes are indentical to exception name without postfix `Exception`, but here not and it sounds too generic. In my mind, it invokes "Hey, create some kind of helper with `camelCaseToKebabCase(removeExceptionPostfix(self::class))` and use it in most implementations of `getUserErrorCode()`". What do you thing about it?

Reply:
Oh, the helper above brings magic and codes will not be searchable :(

Reply:
yeah, it is a good idea and I like it, however, it would probably worsen the code readability and understanding so for now, I will keep it as-is :slightly_smiling_face: